underscore.js

```diff
@@ -1009,6 +1009,7 @@
     if (!_.isObject(obj)) return [];
     var keys = [];
     for (var key in obj) keys.push(key);
+    keys = keys.concat(Object.getOwnPropertySymbols(obj));
```

Comment:
I dont know if we can do this due to <ES5 support

Reply:
This wouldn't necessarily be a problem. We could do feature detection à la `nativeKeys` and transparently ignore symbol keys in environments where they are not supported.